pages/docs/config/models.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Models
+date: "2021-06-22"
+pullQuote: "Learn how to configure apps using Grouparoo's UI Config."
+---
+
+Models are how you define the shape you want your records to have in Grouparoo. Once you've created a model, you can add sources and properties to that model to generate different record types. For example, you may want one model that defines leads, another for customers, and another for site visitors. [See this doc](/docs/getting-started/product-concepts) to learn more about Grouparoo's core concepts.
+
+With [UI Config](/docs/config) running and registered, follow the steps below to configure your model(s).
+
+### Add a Model
+
+Begin by navigating to the Models page from the navigation bar or setup steps page.
+
+<ImageInBrowserFrame
+  centered
+  alt="UI Config: Add Model"
+  src="/images/docs/guides/config/model--01--add.png"
+  width={820}
+  height={402}
+/>
+
+You will see that Grouparoo automatically generates a Profile model for you. You can add a new model by clicking "Add Model".
```

Comment:
https://www.pivotaltracker.com/story/show/179745072 means we will not create a model automatically any more - might as well leave this out